projects/fine_tuning/toolbox/fine_tuning_run_fine_tuning_job/tasks/main.yml

```
@@ -362,3 +362,45 @@ elif fine_tuning_run_fine_tuning_job_workload == 'ilab' -%}
          > {{ artifact_extra_logs_dir }}/artifacts/pod_{{ item | replace('pod/', '') }}.log
     failed_when: false
     loop: "{{ job_pod_name_cmd.stdout_lines }}"
+
+  - name: Retrieve Pod files to artifacts directory
+    when: fine_tuning_run_fine_tuning_job_retrieve_files | bool
+    block:
+      - name: Create the Pod artifacts directory
+        file:
+          path: "{{ artifact_extra_logs_dir }}/pod-artifacts/"
+          state: directory
+          mode: '0755'
+
+      - name: Prepare the retrieval Pod template
+        template:
+          src: "{{ fine_tuning_job_retrieval_pod_template}}"
```

```suggestion
          src: "{{ fine_tuning_job_retrieval_pod_template }}"
```
(I try to always fix the linter warnings, but some of them with multiple lines can't be fixed)